\documentclass[margin,line]{res}


\usepackage{hyperref}
\hypersetup{
   colorlinks,
   citecolor=black,
   filecolor=black,
   linkcolor=black,
   urlcolor=blue
}

\usepackage{url}

\oddsidemargin -.5in
\evensidemargin -.5in
\textwidth=6.0in
\itemsep=0in
\parsep=0in

%if using pdflatex:
\setlength{\pdfpagewidth}{\paperwidth}
\setlength{\pdfpageheight}{\paperheight}

\newenvironment{list1}{
  \begin{list}{\ding{113}}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in}
      \setlength{\leftmargin}{0.17in}}}{\end{list}}
\newenvironment{list2}{
  \begin{list}{$\bullet$}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in}
      \setlength{\leftmargin}{0.2in}}}{\end{list}}


\begin{document}

\name{Enes Kemal Ergin \vspace*{.1in}}

\begin{resume}
\section{\sc Contact Information}
\vspace{.05in}
\begin{tabular}{@{}p{2in}p{4in}}
{\it Cell:} 778-751-1649               & \hfill {\it GitHub:}  \href{https://github.com/eneskemalergin}{eneskemalergin} \\
{\it Work:} 604-875-2000 ext:7146    & \hfill {\it E-mail:}  \href{mailto:eneskemalergin@gmail.com}{eneskemalergin@gmail.com}\\
        & \hfill {\it Portfolio:}  \href{eneskemalergin.github.io}{eneskemalergin.github.io} \\
\end{tabular}


\section{\sc Research Interests}
Computational analysis of proteomic datasets, cancer proteomics, machine learning applications (deep learning in particular) for mass spectrometry data.

\section{\sc Education}

% Uncomment for PhD
{\bf University of British Columbia (UBC)}, Vancouver, BC Canada\\
{\em Bioinformatics}
\begin{list1}
\item[]  PhD, Bioinformatics, 2019 - 2023
\begin{list2}
\vspace*{.05in}
\item {\bf Advisor:}  Philipp Lange
\item {\bf Thesis Title:}  ``Statistical approaches for identifying proteome features in pediatric cancers.''
\end{list2}
\end{list1}

{\bf University of British Columbia (UBC)}, Vancouver, BC Canada\\
{\em Bioinformatics}
\begin{list1}
\item[]  MSc, Bioinformatics, 2017 - 2019(Transfered)
\end{list1}

{\bf North American University (NAU)}, Houston, Texas USA\\
{\em Software Engineering}
\begin{list1}
\item[] BSc, Computer Science,  2013 - 2017
\end{list1}


\section{\sc Research Experience}

{\bf Graduate Research Assistant}

\vspace{-.3cm}
{\em BC Children's Hospital Research Institute} \hfill {\bf September, 2017 - present}\\
\vspace{-.3cm}
\begin{list2}
\item {\bf Research:} Implementing set of statistical and computational methods for exploring proteomic features in childhood cancers based on bottom-up proteomics approaches, with a focus on proteoforms.
\end{list2}

{\bf Visiting Researcher}

\vspace{-.3cm}
{\em Harvard Medical School} \hfill {\bf November, 2015 - September, 2016}\\
\vspace{-.3cm}
\begin{list2}
\item Research: Predicting the determinants of alternative mRNA splicing
\begin{list2}
\item {\bf Aim:} To determine how histone modifications influence the alternative splicing.
\item Developed a {\bf deep convolutional neural network} to predict mRNA expression patterns from 11 different histone modification datasets (pilot in HeLa cells).

\end{list2}
\item Research: Predicting transcription factor binding sites across cell types
\begin{list2}
\item {\bf Aim:} To develop a general framework to predict transcription factor (TF) binding sites accross various cell types.
\item Developed a machine learning-based algorithm can utilize large datasets (approximately 900GB) for estimating TF binding sites.
\end{list2}
\end{list2}

{\bf Undergraduate Research Assistant}

\vspace{-.3cm}
{\em North American University} \hfill {\bf September, 2015 - May, 2017}\\
Leading student in Bioinformatics Lab at NAU. Mentored 4 junior and sophomore students.\\
Designed an open source bioinformatics curriculum with Open Source Society in GitHub.

{\em Yeditepe University} \hfill {\bf August, 2014 - May, 2015}\\
Worked as a genomic data scientist to utilized virtual docking software to determine the best possible inhibitor for specific molecule by mainly using NCBI PubMed database.
%\begin{list2}
%\item
%\end{list2}

{\em Texas Institute of Education and Research (TIBER)} \hfill {\bf September, 2014 - April, 2015}\\
Wet-lab experience: Worked on drug development and testing process. Was responsible for preparing agar solutions, bacteria cultures, and liquid/solid drug tests on those bacteria cultures.


\section{\sc Teaching Experience}
{\bf Teaching Assistant}

\vspace{-.3cm}
{\em North American University} \hfill {\bf September 2015 - May 2017}\\
Co-taught 4 undergraduate level courses for computer science department to over 60 students each semester. Prepared the lab and extra sessions on Git/GitHub, Rapid Python Programming, and Ipython/Jupyter. Shared responsibilities for preparing lectures, exams, homework assignments, and  grading.
\vspace*{.05in}
\begin{list2}
\item COMP 3317 Algorithms, Fall 2015, 2016.
\item COMP 3320 Programming Languates, Fall 2016, Spring 2017.
\item COMP 3322 Software Engineering, Fall 2016.
\item COMP 2415 Systems Programming, Spring 2017
\end{list2}


{\bf Mathematics and Computer Science Tutor}
\vspace{-.1cm}

{\em North American University} \hfill {\bf January - May 2017}\\
Helped and advised students about their school classes. Tutored in various subjects from college algebra to differential equations in Mathematics domain and from CS1 to Data Mining in Computer Science domain.

{\bf Instructor}
\vspace{-.1cm}

{\em North American University} \hfill {\bf February - April 2015}\\
Taught Basic Python programming course to 35 people including students, faculty, and staff of NAU, which was a first ever course taught solely by a student in NAU. (\href{https://github.com/NAU-Python-Class/Py101-Spring-15}{Link})


\section{\sc Publications}
Weng S.H, Demir F., {\bf Ergin E.K}, Dirnberger S., Uzozie A., Tuscher D., Nierves L., Tsui J., Huesgen P.F, Lange P.F. (2019) Sensitive determination of proteolytic proteoforms in limited microscale proteome samples {\em BioRxiv}

Kocabas F., {\bf Ergin E.K}. (2016). Identification of small molecule binding pocket for inhibition of Crimean-Congo hemorrhagic fever virus OTU protease. {\em Turkish Journal of Biology}, 40:239-249.

\section{\sc Posters}
Ji J., Thibodeau M., {\bf Ergin E.K.}, Culibrk L., Smith T.. (2018). SWI/SNF Chromatin Remodeling Complex in Clear Cell Ovarian Cancer. Stat540 Group Project Poster Session.

%\section{\sc Papers in preparation}
%
% Kocabas, F., Ergin, E.K. 2016. Identification of small molecule binding pocket for inhibition of Crimean-Congo hemorrhagic fever virus OTU protease. Turkish Journal of Biology, 40:239-249.

%\section{\sc Conference Presentations}
%
% Kocabas, F., Ergin, E.K. 2016. Identification of small molecule binding pocket for inhibition of Crimean-Congo hemorrhagic fever virus OTU protease. Turkish Journal of Biology, 40:239-249.



\section{\sc Projects}

% Ask this to people and fix descriptions.

{\bf Open Source Bioinformatics Curriculum} \hfill {\bf June  2016}\\
Developed a open source bioinformatics curriculum with contributions of open source society, which gives aspiring bioinformatics scientists chance to start building their foundational knowledge. (\href{https://github.com/open-source-society/bioinformatics}{Link})

{\bf Scholar Development Center} \hfill {\bf February  2016}\\
Created a non-profit community based organization under the Raindrop Foundation to help Turkish undergraduate students around Texas to achieve their dreams in academia or industry.


\section{\sc Honors and Awards}

Michael Cuccione Childhood Cancer Foundation Graduate Studentship, 2018-2019

\vspace*{-2.5mm}
North American University: Exceptional Merit Scholarship, 2012-2017

\vspace*{-2.5mm}
North American University: President's Honor Roll , 2015-2017

\vspace*{-2.5mm}
North American University: Outstanding Student of the Year, 2015


\section{\sc Extracurricular Activities}
\begin{list2}
\item ISCB (International society of computational biology), {\em Member} \hfill {\bf August, 2016 - present}
\item ACM (Association for Computing Machinery), {\em Member} \hfill  {\bf February 2013 - present}
\item NAU Kazakh Student Association, {\em Club Advisor} \hfill {\bf September 2016 - May 2017}
\item NAU Future Leaders Club, {\em Founder and Director} \hfill {\bf April, 2015 - May 2017}
\item Student Government, {\em VP of Unity and Social Justice} \hfill  {\bf September 2014 - May 2017}
\item NAU ACM, {\em Member} \hfill {\bf September 2013 - May 2017}
\item NAU ACM, {\em Vice President} \hfill  {\bf September 2015 - May 2016}
\item NAU ACM, {\em Secretary} \hfill  {\bf February 2013 - September 2014}
\end{list2}

\section{\sc Technical Skills}
\begin{list2}
\item {\bf Languages:}  Python(Pandas, Numpy, H5py, Tensorflow, Biopython, Scikit-learn), R(tidyverse, limma, deseq2, msstats), Java, \LaTeX, C/C++, Shell/Bash Scripting, Javascript, HTML, CSS
\item {\bf Database Systems:} SQL, MySQL, MongoDB
\item {\bf Operating Systems:}  Unix/Linux, MacOS, Windows.
\end{list2}

\section{\sc References}
\begin{list2}
\item Assistant Prof. Dr.Philipp Lange, Pathology Department, University of British Columbia, Vancouver, BC, Canada, \href{philipp.lange@ubc.ca}{philipp.lange@ubc.ca}
\item Assistant Prof. Dr. Stirling L. Churchman, Genetics Department, Harvard Medical School, Boston, MA, USA, \href{churchman@genetics.med.harvard.edu}{churchman@genetics.med.harvard.edu}
\item Associate Prof. Dr. Kemal Aydin, Computer Science Department, North American University, Houston, TX, USA, \href{kemal@na.edu}{kemal@na.edu}
\item Prof. Dr. Cengiz Zubeyir Altuntas, Director of Texas Institute of Biotechnology Education and Research, North American University, Houston, TX, USA \href{cza@na.edu}{cza@na.edu}
\end{list2}


\end{resume}
\end{document}
